app/uk/gov/hmrc/agentsubscription/model/CitizenDetailsMatchResponse.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscription.model
+
+sealed trait CitizenDetailsMatchResponse
+
+object CitizenDetailsMatchResponse {
+  case object Match extends CitizenDetailsMatchResponse
```

Comment:
you can reuse the existing case objects which are generic. 
no need to create new ones for citizen details

Reply:
Have renamed the existing DesMatchResponses to be generic and so useable for CD.